include/gsl/multi_span

```diff
@@ -717,60 +754,59 @@ public:
     using sliced_type = std::conditional_t<rank != 0, strided_bounds<rank - 1>, void>;
     using mapping_type = generalized_mapping_tag;
 
-    constexpr strided_bounds(const strided_bounds&) GSL_NOEXCEPT = default;
+    constexpr strided_bounds(const strided_bounds&) noexcept = default;
 
-    constexpr strided_bounds& operator=(const strided_bounds&) GSL_NOEXCEPT = default;
+    constexpr strided_bounds& operator=(const strided_bounds&) noexcept = default;
 
     constexpr strided_bounds(const value_type (&values)[rank], index_type strides)
         : m_extents(values), m_strides(std::move(strides))
-    {
-    }
+    {}
 
-    constexpr strided_bounds(const index_type& extents, const index_type& strides) GSL_NOEXCEPT
-        : m_extents(extents),
-          m_strides(strides)
-    {
-    }
+    constexpr strided_bounds(const index_type& extents, const index_type& strides) noexcept
+        : m_extents(extents), m_strides(strides)
+    {}
 
-    constexpr index_type strides() const GSL_NOEXCEPT { return m_strides; }
+    constexpr index_type strides() const noexcept { return m_strides; }
 
-    constexpr size_type total_size() const GSL_NOEXCEPT
+    GSL_SUPPRESS(bounds.4) // NO-FORMAT: attribute
+    constexpr size_type total_size() const noexcept
     {
         size_type ret = 0;
-        for (std::size_t i = 0; i < rank; ++i) {
-            ret += (m_extents[i] - 1) * m_strides[i];
-        }
+        for (std::size_t i = 0; i < rank; ++i) { ret += (m_extents[i] - 1) * m_strides[i]; }
```

Comment:
In my opinion the old loop code block was better readable than this one liner.

Reply:
Those are clang-format changes... Normally I would not introduce formatting changes at the same time as code changes, but had to do so in this case (clang-format does not support attributes)